frontend/src/scenes/session-recordings/player/controller/PlayerController.tsx

```diff
@@ -97,9 +99,47 @@ function CinemaMode(): JSX.Element {
     )
 }
 
+function Clip(): JSX.Element {
+    const { takeScreenshot } = useActions(sessionRecordingPlayerLogic)
+
+    return (
+        <LemonButton
+            size="xsmall"
+            onClick={() => takeScreenshot(ExporterFormat.GIF)}
+            tooltip={
+                <>
+                    Get a 5s GIF of this point in the recording{' '}
+                    <LemonTag type="warning" size="small">
+                        BETA
+                    </LemonTag>
+                </>
+            }
+            icon={<IconRecordingClip className="text-2xl" />}
+            data-attr="replay-screenshot-gif"
+            tooltipPlacement="top"
+        />
+    )
+}
+
+function Screenshot(): JSX.Element {
+    const { takeScreenshot } = useActions(sessionRecordingPlayerLogic)
+
+    return (
+        <LemonButton
+            size="xsmall"
+            onClick={() => takeScreenshot(ExporterFormat.PNG)}
+            tooltip="Take a screenshot of this point in the recording"
+            icon={<IconCamera className="text-2xl" />}
+            data-attr="replay-screenshot-png"
```

Comment:
cool that these all have data-attrs
i'm super interested to see if this changes usage (or has usage)

we should also add hot keys for all of these
it's weird that we have one but not others

e.g. we could have

c -> comment
e -> emoji
x -> screenshot
t -> cinema mode (matches youtube)